### DISCUSSION TITLE
要怎麼判斷真笑與假笑

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1552] [General]
法國生理學家杜鄉發現
人在微笑時會動用到兩組關鍵肌肉：
眼角的眼輪匝肌和嘴角的顴肌
所以只要觀察一個地方就能準確判讀
那就是他的眼角有沒有瞇出魚尾紋

我自己平常是只有在尷尬的社交場合才會需要用到假笑。而我覺得如果想要做一個假笑就要做出一個能騙過對方的假笑，別忘記要連你的魚尾紋一起動

你們又是在什麼場合會用到假笑呢？有沒有什麼技巧可以分享的勒

**Comment by poweichen00** [comment]
人有時候會使用假笑來應對各種情況，可能是因為禮貌、社交壓力、尷尬或其他原因，當你感到尷尬或不知所措時，假笑可以是一種應對策略，幫你緩解緊張氛圍。

**Comment by larry40305** [comment]
但要時時刻刻捕捉人類微表情是十分困難的一件事，感覺只有在動漫中才能做到。像在遊戲人生的輕小說裡也有提到微表情，但我不確定輕小說中對微表情的解釋是否正確。

**Comment by aa103967622** [comment]
我覺得從情況就能辨別出他是否在假笑，比如說，在社交場合，比較擅長帶動氣氛的社交咖說話時，說的可能是要讓大家笑一笑，增加氣氛的段子，大概每個人不管好不好笑，都會稍微陪笑一下，基本上，會看氣氛的人都會這樣做。

**Comment by poyen0806** [comment]
我覺得這是一個很有趣的題目
可以用影像識別技術做一個相關的專題
可以用到的函式庫有cv2、tf框架等

也可以考慮使用現成的模型，如 YOLO、Mask R-CNN 等，進一步提升準確度與效率

除此之外，還可以嘗試結合其他技術，如與資料庫或雲端服務整合，實現影像的存取與管理
或透過前端框架設計一個簡單的介面，方便展示成果。這不僅能讓專題更完整，也能展現多方位的技能
